pkgdb/src/registry/floxpkgs.cc

```diff
@@ -631,7 +632,7 @@ getWrappedFlakeNarHash( nix::FlakeRef const & ref )
   nix::StringSink sink;
   nix::dumpPath( flakeDir, sink );
   auto narHash    = hashString( nix::htSHA256, sink.s );
-  auto narHashStr = narHash.to_string( nix::Base16, false );
+  auto narHashStr = narHash.to_string( nix::SRI, true );
```

Comment:
After the schema thing i ran into nix complaining about the nar hash we are expecting against not being in SRI format.